package.json

```diff
@@ -15,8 +15,9 @@
     "minimize": "1.8.x",
     "nunjucks": "1.3.x",
     "nunjucks-markdown": "1.0.x",
-    "raml2obj": "2.2.x",
-    "raml-jsonschema-expander": "1.1.x"
+    "raml2obj": "git://github.com/raml2html/raml2obj.git#raml1.0",
```

Comment:
While this is fine for now, we should really just release a new version of raml2obj, especially if it supports both 0.8 and 1.0. We can even release it as 2.0.0-beta, so that it won't get installed with a regular `npm install raml2obj`.